### PR TITLE
Changes for _include (FHIR-48719, FHIR-48718)

### PR DESCRIPTION
**Pull request by dbojicic**
This PR provides fixes for [FHIR-48718](https://jira.hl7.org/browse/FHIR-48718) and [FHIR-48719](https://jira.hl7.org/browse/FHIR-48719).

### CODE REVIEW

File: input/resources/au-core-requester.xml [hunk 1]
```diff
@@ -874,7 +874,11 @@
 </extension>
 <code value="history-instance"/>
 </interaction>
-<searchInclude value="MedicationRequest:medication"/>
+<searchInclude value="MedicationRequest:medication">
+<extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+<valueCode value="SHALL"/>
```
Comment (JayMurdoch) [inline input/resources/au-core-requester.xml:879]: Change from SHALL to SHOULD

File: input/resources/au-core-responder.xml [hunk 2]
```diff
@@ -866,7 +866,11 @@
 </extension>
 <code value="history-instance"/>
 </interaction>
-<searchInclude value="MedicationRequest:medication"/>
+<searchInclude value="MedicationRequest:medication">
+<extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
+<valueCode value="SHALL"/>
```
Comment (JayMurdoch) [inline input/resources/au-core-responder.xml:871]: Change from SHALL to SHOULD

File: input/resources/au-core-responder.xml [hunk 3]
```diff
@@ -866,7 +866,11 @@
 </extension>
 <code value="history-instance"/>
 </interaction>
-<searchInclude value="MedicationRequest:medication"/>
+<searchInclude value="MedicationRequest:medication">
```
Comment (JayMurdoch) [inline input/resources/au-core-responder.xml:869]: Add blurb to documentation value

**Review by JayMurdoch — APPROVED** [review]

**Review by JayMurdoch — CHANGES_REQUESTED** [review]
See inline comments.

File: input/resources/au-core-requester.xml (hunk 1 above)
Comment (dbojicic) [reply input/resources/au-core-requester.xml:879]: done

File: input/resources/au-core-responder.xml (hunk 2 above)
Comment (dbojicic) [reply input/resources/au-core-responder.xml:871]: Done

File: input/resources/au-core-requester.xml [hunk 4]
```diff
@@ -874,7 +874,11 @@
 </extension>
 <code value="history-instance"/>
 </interaction>
-<searchInclude value="MedicationRequest:medication"/>
+<searchInclude value="MedicationRequest:medication">
```
[the remark this answers is not in the document]
Comment (dbojicic) [reply input/resources/au-core-requester.xml:877]: done

File: input/resources/au-core-responder.xml (hunk 3 above)
Comment (dbojicic) [reply input/resources/au-core-responder.xml:869]: done

File: input/resources/au-core-requester.xml [hunk 5]
```diff
@@ -895,7 +899,7 @@
 <name value="authoredon"/>
 <definition value="http://hl7.org/fhir/SearchParameter/MedicationRequest-authoredon"/>
 <type value="date"/>
-<documentation value="The requester **SHALL** provide a value precise to the *second + time offset*.&#xA;&#xA;The responder **SHALL** support a value precise to the *second + time offset*. &#xA;&#xA;The requester **SHALL** support these search comparators `gt`, `lt`, `ge`, `le`.&#xA;&#xA;The responder **SHALL** support these search comparators `gt`, `lt`, `ge`, `le`.&#xa;&#xa;The requester **SHOULD** support `multipleAnd`, and if `multipleAnd` is supported, **SHALL** support the search comparators `gt`, `lt`, `ge`, `le`.&#xa;&#xa;The responder **SHOULD** support `multipleAnd`, and if `multipleAnd` is supported, **SHALL** support the search comparators `gt`, `lt`, `ge`, `le`."/>
+<documentation value="The requester **SHALL** provide a value precise to the *second + time offset*.&#xA;&#xA;The responder **SHALL** support a value precise to the *second + time offset*. &#xA;&#xA;The requester **SHALL** support these search comparators `gt`, `lt`, `ge`, `le`.&#xA;&#xA;The responder **SHALL** support these search comparators `gt`, `lt`, `ge`, `le`."/>
```
Comment (JayMurdoch) [inline input/resources/au-core-requester.xml:902]: This change is not outlined in the associated Jira tickets.

**Review by JayMurdoch — CHANGES_REQUESTED** [review]
One further comment, thanks.

**Review by JayMurdoch — APPROVED** [review]

**Review by brettesler-ext — APPROVED** [review]

**Pull request merged**